### ISSUE TITLE
Add maximum Y level for random spawn.

### ISSUE THREAD

**Issue by KapoleiMC** [DeathGOD7/unexpectedspawn-paper#26]
I own a Anarchy server and I have players build things within spawn that go all the way up to sky limit and new players that join get stuck up there and can’t make their way down without dying only to be spawned back at the top once again. I know on an anarchy server it’s supposed to be hard to escape spawn. But most new players just end up leaving because there is literally no other option. Please implement this idea of a maximum Y level.

**Comment by KapoleiMC** [comment]
meant to also add that I have expanded the spawn area from 50 blocks to 150 blocks in all directions but it doesn’t matter because it still finds a suitable spawn point for players at 0’0 that ends up being on top of a huge cobblestone sky tower.

**Comment by RexSE7EN** [comment]
Before anything, can you increase the area size in config?
Also, this plugin was designed for random spawn for survival exploration, but the Y-Level feature is also a good idea.

(Note: I don't support anarchy server or HvH server in general but since this feature will be good for survival too, I will look into it)

**Comment by KapoleiMC** [comment]
Yeah I’ve increased the area size but for some reason it still favors spawning the player near 0’0 on top of structures close to sky limit. But if you could look into adding that feature I’d really appreciate it. It would 100% solve the issue I’m having.

**Comment by RexSE7EN** [comment]
Do you have OP bypass permission? Enable the debug log of the plugin.

**Comment by KapoleiMC** [comment]
No I don’t have the op bypass permission under my group in luckperms. But I enabled the debug for for logs.

**Comment by RexSE7EN** [comment]
Can you share the log when it teleports you to the top of 0,0?
